cell_injury.yaml

```diff
@@ -3,9 +3,12 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - python=3.12
+  - python=3.11
+  - pip
   - pandas
   - numpy
   - pre-commit
   - ipykernel
   - requests
+  - pip:
+    - pycytominer
```

Comment:
Why not use the conda-forge implementation? https://anaconda.org/conda-forge/pycytominer

```yaml
- conda-forge::pycytominer==1.1.0
```